README.md

```diff
@@ -359,6 +359,51 @@ container.openPath('game.html');
 any unsupported values. See [the SpringRoll Application Class docs](https://github.com/SpringRoll/SpringRoll/tree/v2/src#handling-state-change)
 for more information on the request format.
 
+### Fullscreen Plugin
+The full screen plugin hooks up a button to set an element to full screen then communicate this through Bellhop. The plugin will also add the class ```'--fullScreen'``` to the button given
+
+```javascript
+import { FullScreenPlugin, Container } from  'springroll-container';
+
+const  container = new  Container('#game', {
+
+  plugins: [
+    // FullScreenPlugin expects the selector for the button to hook onto
+    new  FullScreenPlugin(#fullScreenButton'),
+  ]
+  });
+
+container.openPath('game.html');
+
+```
+
+The plugin will accept either a selector or an array of selectors as a parameter
+
+```javascript
+new  FullScreenPlugin('#fullScreenButton'),
+]
```

Comment:
Weird dangling `]` here. Should we show examples of both a selector and an array of selectors as parameters?